yolo/yolov8_detection.py

```diff
@@ -181,6 +196,23 @@ def webcam():
     vid_cap.release() """
 
 
+# Offline Data loading
+def offline_data(model):
+    # Load the model.
+    model = YOLO('yolov8n.pt')
+
+    # Training.
+    results = model.train(
+        data='C:/Users/frede/Documents/Teamprojekt/teamproject-objectdetection/yolo/data.yaml', # your path
```

Comment:
refactoring necessary ! 
Once again because of its absolute path 
- and maybe to supply a **different training set** by simply changing a variable somewhere, further down in progress!